website/cue/reference/remap/functions/strlen.cue

```diff
@@ -0,0 +1,32 @@
+package metadata
+
+remap: functions: strlen: {
+	category: "Enumerate"
+	description: """
+		Returns the number of UTF-8 characters in `value`. This differs from
```

Comment:
Maybe clarify how we are defining a "character" here, since this isn't a well-defined thing in UTF8. For example, from the `char` type in Rust

> The char type represents a single character. More specifically, since ‘character’ isn’t a well-defined concept in Unicode, char is a ‘Unicode scalar value’, which is similar to, but not the same as, a ‘Unicode code point’.

https://doc.rust-lang.org/stable/std/primitive.char.html

Reply:
👍 added a note in b7474411b